website/client/components/guilds/groupFormModal.vue

```diff
@@ -0,0 +1,285 @@
+      font-weight: bold;
+      margin-right: 1em;
+      color: #24cc8f;
+    }
+  }
+
+  .description-count {
+    margin-top: 1em;
+  }
+
+  .category-select {
+    border-radius: 2px;
+    background-color: #ffffff;
+    box-shadow: 0 2px 2px 0 rgba(26, 24, 29, 0.16), 0 1px 4px 0 rgba(26, 24, 29, 0.12);
+    padding: 1em;
+  }
+
+  .category-select:hover {
+    cursor: pointer;
+  }
+
+  .category-wrap {
+    margin-top: .5em;
+  }
+
+  .icon {
+    margin-left: .5em;
+    display: inline-block;
+  }
+</style>
+
+<script>
+import bModal from 'bootstrap-vue/lib/components/modal';
+import bBtn from 'bootstrap-vue/lib/components/button';
+import bFormInput from 'bootstrap-vue/lib/components/form-input';
+import bFormCheckbox from 'bootstrap-vue/lib/components/form-checkbox';
+import bFormSelect from 'bootstrap-vue/lib/components/form-select';
+import bTooltip from 'bootstrap-vue/lib/components/tooltip';
+
+export default {
+  components: {
+    bModal,
+    bBtn,
+    bFormInput,
+    bFormCheckbox,
+    bFormSelect,
+    bTooltip,
+  },
+  data () {
+    let data = {
+      newGuild: {
+        name: '',
+        type: 'guild',
+        privacy: 'private',
+        description: '',
+        categories: [],
+        onlyLeaderCreatesChallenges: true,
+        guildLeaderCantBeMessaged: true,
+        privateGuild: true,
+        allowGuildInvationsFromNonMembers: true,
+      },
+      categoryOptions: [
+        {
+          label: 'animals',
+          key: 'animals',
+        },
+        {
+          label: 'artDesign',
+          key: 'art_design',
+        },
+        {
+          label: 'booksWriting',
+          key: 'books_writing',
+        },
+        {
+          label: 'comicsHobbies',
+          key: 'comics_hobbies',
+        },
+        {
+          label: 'diyCrafts',
+          key: 'diy_crafts',
+        },
+        {
+          label: 'education',
+          key: 'education',
+        },
+        {
+          label: 'foodCooking',
+          key: 'food_cooking',
+        },
+        {
+          label: 'healthFitness',
+          key: 'health_fitness',
+        },
+        {
+          label: 'music',
+          key: 'music',
+        },
+        {
+          label: 'relationship',
+          key: 'relationship',
+        },
+        {
+          label: 'scienceTech',
+          key: 'science_tech ',
+        },
+      ],
+      showCategorySelect: false,
+    };
+
+    let hashedCategories = {};
+    data.categoryOptions.forEach((category) => {
+      hashedCategories[category.key] = category.label;
+    });
+    data.categoriesHashByKey = hashedCategories;
+
+    return data;
+  },
+  computed: {
+    charactersRemaining () {
+      return 500 - this.newGuild.description.length;
+    },
+  },
+  methods: {
+    toggleCategorySelect () {
+      this.showCategorySelect = !this.showCategorySelect;
+    },
+    async submit () {
+      if (this.$store.state.user.data.balance < 1) {
+        alert('Not enough gems');
```

Comment:
let's add a TODO so we'll remember to use proper notifications once we have the designs